pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -170,7 +170,7 @@ func RunListBuildWebHooks(f *clientcmd.Factory, out, errOut io.Writer, name stri
 		}
 		ref := build.Config
 		if ref == nil {
-			return fmt.Errorf("the provided build %q was not created from a build config and cannot have webhooks", name)
+			return fmt.Errorf("the provided build %q was not created from a buildConfig and cannot have webhooks", name)
```

Comment:
I'm not sure these really qualify as "end user messages". This is second deepest level of verbose tracing.  I'd think some flexibility would be acceptable at this level. 

Ben Parees | OpenShift

From: Clayton Coleman <notifications@github.com>
Sent: May 22, 2015 11:41 PM
To: openshift/origin
Cc: Ben Parees
Subject: Re: [origin] Improving/Adding logging messages (#2452)

> @@ -170,7 +170,7 @@ func RunListBuildWebHooks(f *clientcmd.Factory, out, errOut io.Writer, name stri
>  		}
>  		ref := build.Config
>  		if ref == nil {
> -			return fmt.Errorf("the provided build %q was not created from a build config and cannot have webhooks", name)
> +			return fmt.Errorf("the provided build %q was not created from a buildConfig and cannot have webhooks", name)

build configuration - for end user messages we should never use "BuildConfig" or "DeploymentConfig", but instead "build configuration" or "deployment configuration".  "build config" or "deployment config" is acceptable in a paragraph where you've already used "build configuration".

---
Reply to this email directly or view it on GitHub:
https://github.com/openshift/origin/pull/2452/files#r30941619